### ISSUE TITLE
TEI/Epidoc : //note/lemma is incorrect.

### ISSUE THREAD

**Issue by PonteIneptique** [PerseusDL/canonical#147]
[Example](https://github.com/PerseusDL/canonical/blob/master/CTS_XML_TEI/perseus/latinLit/phi0959/phi008/phi0959.phi008.perseus-lat1.xml)

What should we do with //note/lemma ? Transform it to //note/w@lemma="text"

**Comment by simonastoyanova** [comment]
Or change <note> to <app> and then have the lemma in <lem>?

**Comment by lcerrato** [comment]
Is it the example here?
<l n="5">iam prope lux aderat, qua <note id="note.20.1"><lemma>qua</lemma> cum</note> me discedere Caesar</l>
Looks like a variant with <lemma> being used to designate that the "qua" has a variant of "qua cum"

**Comment by PonteIneptique** [comment]
Bumping this issue again, as I found another one and we did not give an answer I think.
It's exactly what you said @lcerrato , this is a variant. The question is how to transform the note into something more usable : 

In phi0474.phi024.perseus-lat1.xml

``` xml
quod
<note>
  <lemma>quod</lemma> quia 
  <hi rend="italics"><foreign xml:lang="grc">ς</foreign>B</hi>
</note> diebus festis ludisque publicis, omnibus forensibus negotiis intermissis, u
```

I assume we would go with the [Inline apparatus criticus](http://www.stoa.org/epidoc/gl/latest/supp-app-inline.html)

I would propose : 

``` xml
quod <note>
 <app type="alternative">
    <lem>quod</lem>
    <rdg wit="ς">quia</rdg>
 </app>
</note>
```

or 

``` xml
 <app type="alternative">
    <lem>quod</lem>
    <rdg wit="ς">quia</rdg>
 </app>
```

**Comment by PonteIneptique** [comment]
Ping @lcerrato @simonastoyanova

**Comment by balmas** [comment]
I like the idea of using the app crit markup

**Comment by lcerrato** [comment]
Sounds like a plan.

**Comment by simonastoyanova** [comment]
Your guesses look good to me. Thinking of the Perseus CSS, of which I don't know anything, does it matter if we put the "quod" in the app/lem only (option 2), or if we leave it in the text as well (option 1)? Personally I'd prefer option 2, I think, which doesn't repeat the word. Otherwise it's all fine!

**Comment by PonteIneptique** [comment]
Thanks all of you !

**Comment by lcerrato** [comment]
@simonastoyanova I don't think it would be duplicated here. Looks like it is just an illustration to show that the note is attached to the quod. (Unless I'm reading it wrong). It's a matter of whether you want a note or inline in a case where the original offers no precedent. If there's a note in the original text, I would keep the note. But I'm willing to be convinced otherwise.
But I don't see the current Perseus implementation having any bearing on the ultimate decision as this won't appear in P4 so current Perseus practice in terms of display is not a deciding factor.

**Comment by simonastoyanova** [comment]
Thanks, this answers my question. Of course, we keep the note if it’s there.

On Thursday, 29 October 2015, Lisa Cerrato notifications@github.com wrote:

> @simonastoyanova I don't think it would be duplicated here. Looks like it
> is just an illustration to show that the note is attached to the quod.
> (Unless I'm reading it wrong). It's a matter of whether you want a note or
> inline in a case where the original offers no precedent. If there's a note
> in the original text, I would keep the note. But I'm willing to be
> convinced otherwise.
> But I don't see the current Perseus implementation having any bearing on
> the ultimate decision as this won't appear in P4 so current Perseus
> practice in terms of display is not a deciding factor.
> 
> —
> Reply to this email directly or view it on GitHub.<
> https://ci5.googleusercontent.com/proxy/zFWKpwfFKD2ElnCmhMTcU9CU-Z16B3KeyQpV59iYDE7To_drc9xk-MhWFYwtqxUkIS75ubP307smWcej4PU64HvAvQoT6RuW2ZCsQ7OahyPsTIv7jl3AK-Wj49uJQohseNeLnp1_g2YYV_RV5vgwZlqTXoiQiQ=s0-d-e1-ft#https://github.com/notifications/beacon/AEtm5yhq7LNGhdNkP8-IPNYONEzn9xP7ks5pAjp-gaJpZM4EtRBM.gif

## 

Simona Stoyanova
Research Associate

Digital Humanities
Department of Computer Science
University of Leipzig
Augustusplatz 10
04109 Leipzig, Germany

**Comment by PonteIneptique** [comment]
In this context, and the fact that actually, the use of  `<note>` is far from being harmonized, I went with `<note><app><lem>hospitiisque</lem></app>..`

Automatic treatment is far too complex to be dealt with. This way I don't lose information, achieve epidoc compliancy and we can look at it later for transforming the note into a real critical apparatus.
